### PR TITLE
Fixed input component change event

### PR DESCRIPTION
**Pull request by wachterjohannes**
Main PR: https://github.com/sulu-io/sulu/pull/1800

### CODE REVIEW

**Comment by danrot** [comment]
Why is this necessary?

**Comment by wachterjohannes** [comment]
because for the preview we have to fire this update event from the container of a component because only on this the preview will listen

File: husky_components/input/main.js [hunk 1]
```diff
@@ -200,6 +200,13 @@ define([], function() {
          * Binds Dom-related events
          */
         bindDomEvents: function() {
+            this.sandbox.dom.on(this.$el, 'change', function(event) {
+                event.stopPropagation();
+                event.preventDefault();
+
+                this.$el.trigger('change');
```
Comment (danrot) [inline husky_components/input/main.js]: Referring to your other comment: What you are basically doing here is to catch the event emitted on `this.$el`, and triggering again, so how can this improve the situation?

Comment (wachterjohannes) [inline husky_components/input/main.js]: you have overseen the filter in the next line (: i catch the event from the input not from the $el!

**Pull request merged**